src/HTTPSsupportDropdown.js

```diff
@@ -1,22 +1,33 @@
 import React, { useContext } from "react";
 import { searchContext } from "./search/searchContext";
+import "./HTTPSsupportDropdown.css";
 
 export default function HTTPSsupportDropdown() {
   const { setHTTPSsupport } = useContext(searchContext);
-  const onChange = (e) => setHTTPSsupport(e.target.value || null);
-  const allHTTPS = ["true", "false"];
+  const onClick = (e) => {
+    let content = e.target.textContent;
+    if (content.length > 0) {
+      if (content === "HTTPS") {
+        content = "True"
+      } else if (content === "HTTP") {
+        content = "False"
+      } else {
+        content = null
+      }
+      setHTTPSsupport(content || null);
```

Comment:
no need for `content || null` as content will always be null

Reply:
fixed it